Cargo.toml

```diff
@@ -14,13 +14,13 @@ description = "A Rust version of ttrpc."
 protobuf = { version = "2.0", optional = true }
 bytes = { version = "0.4.11", optional = true }
 libc = { version = "0.2.59", features = [ "extra_traits" ] }
-nix = "0.16.1"
+nix = "^0.20.2"
```

Comment:
The '^' is redundant, cargo treats versions that do not have any operators as they have '^'.
Although it can be deleted or not, but for consistency reasons I think it should be deleted

Reply:
Fixed.